src/platform_projects/openthread-efr32-soc-with-buttons.slcp

```diff
@@ -0,0 +1,51 @@
+
+project_name: openthread-efr32-soc-with-buttons
+label: ot-efr32
+package: OpenThread
+description: This is the platform project that will enable the ot-efr32 repo on GitHub to generate a CMake library for EFR32
+category: OpenThread Examples
+quality: production
+
+component:
+  - id: brd4001a
```

Comment:
We should probably remove brd4001a from all the slcp files but maybe we do that when we add support for GSDK 4.1.2.

Reply:
I'll do that as part of the GSDK 4.1.2 update